guides/RestAPIGuide.md

```diff
@@ -1024,45 +1024,45 @@ Note that the name and description fields are read-only.  If you post an item ob
 
 
 ##### Attachment
-```
+```json
 {
-  "uuid" : <uuid>,
-  * "type" : <string>,
-  * "description" : <string>,
-  "viewer" : <string>,
-  "preview" : <boolean>
+  "uuid" : "<uuid>",
+  * "type" : "<string>",
+  * "description" : "<string>",
+  "viewer" : "<string>",
+  "preview" : "<boolean>"
```

Comment:
Shouldn't need to quote booleans.

Reply:
https://github.com/equella/equella.github.io/pull/25#discussion_r234487858